test.py

```diff
@@ -0,0 +1,38 @@
+import pandas as pd
```

Comment:
Was this file meant to be committed?

Reply:
No, sorry, that was a test script hanging out in that directory - that's what I get for doing a `git add *`